src/treestatus/api/treestatus_api/api.py

```diff
@@ -36,7 +51,184 @@ def _now():
     return datetime.datetime.utcnow().replace(tzinfo=pytz.UTC)
 
 
+def _statuspage_data(
+    close,
```

Comment:
Can you rename `close` to `resolve` please. We usually use `close` as an action to close a tree.

Reply:
i will be more specific and use `tree_closed`

Reply:
i will just use resolved as you suggested